frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -619,6 +619,14 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
   props,
 ) => {
   const { spec, metadata, status } = props.obj;
+  const ibmSupportLink = _.get(props.obj, [
+    'status',
+    'channels',
+    '0',
+    'currentCSVDesc',
+    'annotations',
+    'marketplace.openshift.io/support-workflow',
```

Comment:
@alimobrem @robszumski It looks like there's an existing `support` annotation. Is there a reason we can't simply use that?

Reply:
We are using that one, right?